_dev/docker/builder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:12-slim
+FROM node:12.16.3-slim
```

Comment:
Hrm. Was this change intentional for some reason?

Reply:
though moot at this point, since we aren't releasing this, the build error in payments server was triggered by node 12.17